src/store/wallets/wallets.utils.js

```diff
@@ -0,0 +1,19 @@
+export const walletConnectV2Connector = async (EthereumProvider, options) => {
```

Comment:
```suggestion
export const walletConnectV2Connector = async (_package, options) => {
```
I would avoid naming an argument as a class (using same case).
Choose `_package` or `ethereumProvider`